### ISSUE TITLE
Emty slot in NMS save editor

### ISSUE THREAD

**Issue by tjah007** [goatfungus/NMSSaveEditor#412]
hi, my old saves were bugged (2018) so made a stone from my heart and deleted every save and starting over.
started "normal" game and after intro i tried the save editor.. i have one slot, but it's emty
did lot's off reading and made a new game on "creative" just to be sure that the file path was correct (and it is)
played for a while and shut down the game. i opend the editor again, guess what, now i have 2 emty slots.

if i go to the folder just browsig my files, i can see the 2 saves i made with correct time and date. if i go to that folder in the editor, all i find are emty slots.

i've included the log it created. hope you can help me. 
i'm a noob that loves playing space games.

so many, many thx for the editor and all the work that you have put in  :D

[NMSSaveEditor.log](https://github.com/goatfungus/NMSSaveEditor/files/7111652/NMSSaveEditor.log)

**Comment by Vypir** [comment]
Hey Tjah, if you check out some of the other threads here, you'll see that NMSSaveEditor isn't working with the latest update for NMS (Frontiers).  Goatfungus is working on it and we need to be patient (just telling myself this over and over).
I did the same thing you did and have the empty slots as well.
Maybe goat will be enticed to work faster if we donate more to his patreon page. lol

**Comment by tjah007** [comment]
> Hey Tjah, if you check out some of the other threads here, you'll see that NMSSaveEditor isn't working with the latest update for NMS (Frontiers). Goatfungus is working on it and we need to be patient (just telling myself this over and over).
> I did the same thing you did and have the empty slots as well.
> Maybe goat will be enticed to work faster if we donate more to his patreon page. lol

Vypir,
Seems like I misted those treads :p or I was looking for the wrong info...
searched a lot on empty slot and file path not ok..
makes me regret that I deleted my 400h save :p 
pff doesn't matter anymore once the editor is back online..
didn't do much with it after all. gave myself some credits and S class fighter.
then I played the game. (I always want to start with something good) makes
the game a bit easier for me.

anyway; thx for replying!! 
I'll just wait it out then like all the other players.
Donating... I usually don't do that. 
I do it with 1 youtuber and it's just buying coffee 
so almost every month I buy him like 2 or 3 coffees. 
but I will look on his page for donating something small like 5 euro or so.. doesn't bother me. 
But first the fix  lmao :p

Thx again.
